src/9.negotiation.js

```diff
@@ -607,8 +607,13 @@ export function /*9.2.8 */SupportedLocales (availableLocales, requestedLocales,
             writable: false, configurable: false, value: subset[P]
         });
     }
-    // "Freeze" the array so no new elements can be added
-    defineProperty(subset, 'length', { writable: false });
+    
+    // Wrap in try catch for older browsers that don't support setting length of
```

Comment:
Fixed the lint errors and ran the tests. All of the unit tests passed but I did get a failure when trying to run Sauce Labs.
```
throw new Error('Could not start Sauce Tunnel');
```
More than happy to update my MR or implement an alternative.